langchain/chains/llm.py

```diff
@@ -43,6 +48,8 @@ def lc_serializable(self) -> bool:
     """Prompt object to use."""
     llm: BaseLanguageModel
     output_key: str = "text"  #: :meta private:
+    output_parser: BaseLLMOutputParser = Field(default_factory=DefaultOutputParser)
```

Comment:
We should document that the default parser is a no-op / passthrough parser (alternatively, we could use a None to represent no-op)